cli/template/base/_eslintrc.cjs

```diff
@@ -24,6 +24,7 @@ const config = {
       },
     ],
     "@typescript-eslint/no-unused-vars": ["warn", { argsIgnorePattern: "^_" }],
+    "@typescript-eslint/require-await": "off",
```

Comment:
added this cause this rule is annoying in RSC since for example server actions have to be async, leading to a lot of `disable`'s for this rule spread around